### PR TITLE
fix(website): fix billing FAQ for Team plan

### PR DESCRIPTION
**Pull request by jamilbk**

### CODE REVIEW

File: website/src/app/pricing/_page.tsx [hunk 1]
```diff
@@ -371,8 +371,8 @@ export default function _Page() {
             <p>When will I be billed?</p>
           </blockquote>
           <p className="mb-8">
-            When you start service, or at the beginning of each billing cycle.
-            Enterprise plans are billed quarterly or annually.
+            The Team plan is billed when you start service, or at the beginning
+            of each billing cycle. Enterprise plans are billed annually.
```
Comment (bmanifold) [inline website/src/app/pricing/_page.tsx]: One question that comes to my mind is, "is the first cycle prorated if I start part way through a cycle?".  Also wondering if "billing cycle" is defined anywhere?

Comment (jamilbk) [reply website/src/app/pricing/_page.tsx]: good catch updated

File: website/src/app/pricing/_page.tsx [hunk 2]
```diff
@@ -371,8 +371,8 @@ export default function _Page() {
             <p>When will I be billed?</p>
           </blockquote>
           <p className="mb-8">
-            When you start service, or at the beginning of each billing cycle.
-            Enterprise plans are billed quarterly or annually.
+            The Team plan is billed when you start service, and then monthly
```
Comment (bmanifold) [inline website/src/app/pricing/_page.tsx:374]: I like the billing cycle being defined.  Just wondering if the first month is prorated or if the customer will pay for the whole month no matter when they start on the team plan?

**Review by bmanifold — APPROVED** [review]
LGTM.  Just one question that's not necessarily blocking.

File: website/src/app/pricing/_page.tsx (hunk 2 above)
Comment (jamilbk) [reply website/src/app/pricing/_page.tsx:374]: Gotcha, I see. Will open another Pr.

**Pull request merged**